### PR TITLE
update fuction/property names after moose update

### PR DESCRIPTION
**Pull request by BoZeng1997**
update some object parameter names, function names so that they stop throwing warnings. Changes includes:

The RankTwoTensor initializer in PhaseFieldJIntegral
The "f_name" to "property_name" in CrackGeometricFunction and DegradationFunctionBase

### CODE REVIEW

**Comment by BoZeng1997** [comment]
more update based on moose 01a4027
1. add namespace std to pow function in Nucleation model objects
2. update getValue() function type to const

**Comment by BoZeng1997** [comment]
for MOOSE 7a54ed3 (Thu Aug 24 14:28:21), the update on const virtual function getValue() will hit compile error on dbg mode

File: include/bcs/DashpotPiecewiseBC.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+#pragma once
+
+#include "IntegratedBC.h"
+
+// Forward Declarations
+
+/**
+ * Implements a simple constant Dashpot BC where grad(u)=value on the boundary.
+ * Uses the term produced from integrating the diffusion operator by parts.
```
Comment (hugary1995) [inline include/bcs/DashpotPiecewiseBC.h:18]: This explanation doesn't make sense to me. How is this related to the diffusion operator?

File: include/bcs/DashpotPiecewiseBC.h [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ */
+class DashpotPiecewiseBC : public IntegratedBC
+{
+public:
+  static InputParameters validParams();
+
+  /**
+   * Factory constructor, takes parameters so that all derived classes can be built using the same
+   * constructor.
+   */
```
Comment (hugary1995) [inline include/bcs/DashpotPiecewiseBC.h:28]:
```suggestion
```

File: include/bcs/DashpotPiecewiseBC.h [hunk 3]
```diff
@@ -0,0 +1,50 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
```
Comment (hugary1995) [inline include/bcs/DashpotPiecewiseBC.h:8]: The license header should use the RACCOON one.

File: src/bcs/DashpotPiecewiseBC.C [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      "coeff_tension", 1.0, "The viscosity coefficient when spring is in tension");
+  params.addParam<Real>(
+      "coeff_compression", 1.0, "The viscosity coefficient when spring is in compression");
+  return params;
+}
+
+DashpotPiecewiseBC::DashpotPiecewiseBC(const InputParameters & parameters)
+  : IntegratedBC(parameters),
+    _component(getParam<unsigned int>("component")),
+    // _coefficient(getParam<Real>("coefficient")),
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:35]:
```suggestion
```

File: src/bcs/DashpotPiecewiseBC.C [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+DashpotPiecewiseBC::validParams()
+{
+  InputParameters params = IntegratedBC::validParams();
+  params.addRequiredParam<unsigned int>(
+      "component", "The displacement component corresponding the variable this BC acts on.");
+  params.addRequiredCoupledVar("disp_x", "Displacement in the x direction");
+  params.addCoupledVar("disp_y", "Displacement in the y direction");
+  params.addCoupledVar("disp_z", "Displacement in the z direction");
+
+  // params.addParam<Real>("coefficient", 1.0, "The viscosity coefficient");
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:24]:
```suggestion
```

File: src/bcs/DashpotPiecewiseBC.C [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "DashpotPiecewiseBC.h"
+
+registerMooseObject("raccoonApp", DashpotPiecewiseBC);
+
+InputParameters
+DashpotPiecewiseBC::validParams()
+{
+  InputParameters params = IntegratedBC::validParams();
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:17]: params.addClassDescription(...)

File: src/bcs/DashpotPiecewiseBC.C [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  params.addRequiredParam<unsigned int>(
+      "component", "The displacement component corresponding the variable this BC acts on.");
+  params.addRequiredCoupledVar("disp_x", "Displacement in the x direction");
+  params.addCoupledVar("disp_y", "Displacement in the y direction");
+  params.addCoupledVar("disp_z", "Displacement in the z direction");
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:22]: Why not params.addCoupledVar("displacements", ...)?

File: src/bcs/DashpotPiecewiseBC.C [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+  // params.addParam<Real>("coefficient", 1.0, "The viscosity coefficient");
+  params.addParam<Real>(
+      "coeff_tension", 1.0, "The viscosity coefficient when spring is in tension");
+  params.addParam<Real>(
+      "coeff_compression", 1.0, "The viscosity coefficient when spring is in compression");
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:28]: We don't say a dashpot is in tension/compression.

File: src/bcs/DashpotPiecewiseBC.C [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+  if (_normals[_qp] * velocity > 0)
+    // {
+    _coefficient = _coeff_compression;
+  // std::cout<<"_coeff_compression "<<_coefficient<<std::endl;}
+  else
+    _coefficient = _coeff_tension;
+
+  return _test[_i][_qp] * _coefficient * _normals[_qp] * velocity;
+}
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:61]: Where is `component`?

File: src/bcs/DashpotPiecewiseBC.C [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    component = 0;
+  else if (jvar == _disp_y_var)
+    component = 1;
+  else if (jvar == _disp_z_var)
+    component = 2;
+
+  velocity(component) = _phi[_j][_qp] / _dt;
+
+  return -_test[_i][_qp] * _normals[_qp] * velocity;
+}
```
Comment (hugary1995) [inline src/bcs/DashpotPiecewiseBC.C:93]: This doesn't look right.

**Review by hugary1995 — CHANGES_REQUESTED** [review]

**Comment by BoZeng1997** [comment]
I am so sorry. The DashpotPiecewiseBC should have not been committed in the first place. I just removed related files. But thanks for the advice.

**Comment by hugary1995** [comment]
Please base against devel, not master.

**Pull request closed without merging**